kimchi/src/verifier.rs

```diff
@@ -616,7 +627,11 @@ where
                         panic!("runtime lookup table is unused in the linearization")
                     }
                     Index(t) => {
-                        use GateType::*;
+                        use GateType::{
+                            CairoClaim, CairoFlags, CairoInstruction, CairoTransition, ChaCha0,
+                            ChaCha1, ChaCha2, ChaChaFinal, CompleteAdd, EndoMul, EndoMulScalar,
+                            Generic, Lookup, Poseidon, RangeCheck0, RangeCheck1, VarBaseMul, Zero,
+                        };
```

Comment:
not a fan of that change also, I find that in general it is fine to open with a wildcard if it's an enum and right before using the enum